packages/data/package.json

```diff
@@ -23,7 +23,8 @@
 	"dependencies": {
 		"@babel/runtime-corejs2": "7.12.5",
 		"@woocommerce/date": "2.1.0",
-		"@woocommerce/navigation": "5.2.0"
+		"@woocommerce/navigation": "5.2.0",
+		"rememo": "^3.0.0"
```

Comment:
Added `rememo` as mentioned in the PR description (same as what wordpress/core-data uses).